accepted/00000-hub-reporting.md

```diff
@@ -0,0 +1,208 @@
+- Feature Name: Hub Reporting
+- Start Date: 2021-11-23
+
+# Summary
+[summary]: #summary
+
+Design and implement the infrastructure for a central reporting on Uyuni Hub.
+
+# Motivation
+[motivation]: #motivation
+
+In a Uyuni Hub scenario we have the Hub to prepare and provide content for multiple other Uyuni Servers.
+The goal is now to get data from these Servers back and have combined reporting data available on the Hub.
+
+The data should be made available for external Reporting Tools.
+
+If possible, the data should also be usable on a single Uyuni Server with the same mechanism.
+
+
+# Detailed design
+[design]: #detailed-design
+
+The data should be made available in a PostgreSQL database. We considered using other database types,
+but the increased maintenance effort for new packages and limited knowledge about NoSQL databases
+let us stay with the good well known PostgreSQL database.
+Also PostgreSQL seems to be accepted in most Reporting tools as data source, while NoSQL databases
+are not supported, or only via standard DBMS modules. Native support was rarely available.
+
+### The Database
+The main database is a PostgresDB in the Uyuni Hub system (but in the future it will be possible to 
+use also an external db): it stores all the information collected from all the server, and eventually
+aggregates them. Other databases with the same schema are also present in all the server, to collect 
+information for that system.
+All the databases (in the Uyuni Hub and in all Uyuni Server) needs to be made available on the network to either connect in a secure way (using the SSL
+certificates provided by Uyuni Server) with the reporting tool or (for Uyuni Server) with the Hub to gather the data. 
+If possible the connection to the main Uyuni DB from the outside should be forbidden.
+In summary:
+- the DB in Uyuni Hub stores, collects and eventually aggregates data coming from all the DBs of the managed Uyuni Server,
```

Comment:
```suggestion
- the DB in Uyuni Hub stores, collects and eventually aggregates data coming from all the DBs of the peripheral Servers,
```

For consistency's sake with existing documents, can you refer to Servers registered to a Hub as "peripheral Servers" throughout this RFC please?